scripts/changelog_check.rb

```diff
@@ -99,7 +112,7 @@ def generate_changelog(base_branch, source_branch)
       category: category,
       subcategory: change[:subcategory].capitalize,
       pr_number: pr_number&.named_captures&.fetch('pr'),
-      change: change[:change].capitalize,
+      change: change[:change].sub(/./, &:upcase),
```

Comment:
Could we add a fixture case for the uppercase formatting?

Reply:
added a spec in [806ba4b](https://github.com/18F/identity-idp/pull/6031/commits/806ba4b124927ccfd7198e1c229161c3464333d3)